lib/features/base/base_controller.dart

```diff
@@ -168,6 +160,12 @@ abstract class BaseController extends GetxController
 
   void handleExceptionAction({Failure? failure, Exception? exception}) {
     logError('BaseController::handleExceptionAction():failure: $failure | exception: $exception');
+    if (!authorizationInterceptors.isAppRunning) {
+      return;
+    }
+    if (exception is BadCredentialsException || exception is ConnectionError) {
```

Comment:
Why need to log out when we have `ConnectionError`?

Reply:
It is one of the cases we tested before. When response is null